perceval/utils/parameter.py

```diff
@@ -70,8 +69,8 @@ def __init__(self, name: str, value: float = None,
         self._periodic = periodic
         self._pid = Parameter._id
         self._is_expression = is_expression
```

Comment:
This could be set as property, like
```
@property
def _is_expression(self):
    return not type(self) == Parameter
```
This would avoid having to give this as argument